constraint/pkg/client/regolib/src.go

```diff
@@ -4,8 +4,14 @@ const (
 	targetLibSrc = `
 package hooks["{{.Target}}"]
 
+# Auto reject reviews when a constraint has namespaceSelector but Namespace is not cached
+deny[response] {
+	data.hooks["{{.Target}}"].library.autoreject_review[response]
+}
+
 # Finds all violations for a given target
 deny[response] {
+	not data.hooks["{{.Target}}"].library.autoreject_review[response]
```

Comment:
this `not` line should be dropped. We should keep reporting violations even if autoreject is also true as the intent is to return as much actionable feedback (violations) as possible.